tests-e2e/util/index.ts

```diff
@@ -1,4 +1,5 @@
 export { default as CoursePage } from "./CoursePage"
+export { default as VideoSection } from "./VideoSection"
 export { default as WwwPage } from "./WwwPage"
```

Comment:
Ahh, the recently added VideoElement did not get added to these exports, or you probably would have noticed it. Oops.